src/mappings/entities/mapStaking.ts

```diff
@@ -19,8 +19,10 @@ const bondedUnbondedOrReward = new Set([
   StakingEventType.Unbonded,
   StakingEventType.Reward,
 ]);
-
-export async function handleStakingEvent(
+/**
+ * Subscribes to events related to staking events
+ */
+export async function mapStaking(
```

Comment:
The entity is a StakingEvent

Reply:
My bad, when I was doing funding I saw these and though I was supposed to add Event to the name.

Had another look at tooling and see my mistake now.